packages/engine/src/scene/functions/loaders/ModelFunctions.ts

```diff
@@ -66,6 +68,15 @@ export const updateModel: ComponentUpdateFunction = (entity: Entity, properties:
   if (typeof properties.textureOverride !== 'undefined') {
     overrideTexture(entity)
   }
+
+  if (properties.useBasicMaterial !== undefined) {
```

Comment:
this should be `if (typeof properties.useBasicMaterial == 'boolean' && properties.useBasicMaterial) {`

Reply:
This is equivalent, and the next line is required for toggle functionality